xrviz/dashboard.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, data):
         if not isinstance(data, xr.core.dataarray.DataWithCoords):
             raise ValueError("Input should be an xarray data object, not %s" % type(data))
         self.set_data(data)
+        self.cartopy_installed()
```

Comment:
Yeah, so this makes duplication: would be better to have the import logic in utils module, and import, rather than repeatedly calling methods.

Reply:
I have tried but not able to figure out a way to import `cartopy`, `geoviews` in utils and make these availble globally in dashboard. So `cartopy_geoviews_installed` is defined in dashboard and `has_cartopy` is also now global in dashboard.